clicommand/agent_start.go

```diff
@@ -62,6 +62,7 @@ type AgentStartConfig struct {
 	DisconnectAfterIdleTimeout  int      `cli:"disconnect-after-idle-timeout"`
 	BootstrapScript             string   `cli:"bootstrap-script" normalize:"commandpath"`
 	CancelGracePeriod           int      `cli:"cancel-grace-period"`
+	EnableJobLogFile            bool     `cli:"enable-job-log-file"`
```

Comment:
Could we change this (and all its dependent bits and bobs, the flag and envar etc) to something like
```suggestion
	JobLogLocalCache            bool     `cli:"job-log-local-cache"`
```
i'm very much open to suggestions on naming here, but i want to make it really clear to casual readers of the docs that the log file that this flag enables is ephemeral, and won't stick around after the job is complete - i'm keen to have a word like "cache" or "ephemeral" or "temporary" or something in there.

Reply:
Yeah perhaps `--enable-job-log-tmpfile` or similar?

Reply:
Sounds good, renaming to `--enable-job-log-tmpfile` to make the naming more explicit